package.json

```diff
@@ -17,12 +16,10 @@
   "homepage": "https://github.com/JAMS-Front-End-Capstone/module1#readme",
   "dependencies": {
     "axios": "^0.21.1",
-    "bluebird": "^3.7.2",
     "body-parser": "^1.19.0",
-    "cors": "^2.8.5",
     "express": "^4.17.1",
-    "moment": "^2.29.1",
     "mongoose": "^5.11.12",
+    "morgan": "^1.10.0",
     "nodemon": "^2.0.7",
```

Comment:
nodemon would be a dev dependency.  We don't want the server restarting itself every time there's an error in production. We should handle those errors, or crash it.